src/test/java/BoardCreatorTest.java

```diff
@@ -1,6 +1,8 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
+import engine.entities.*;
```

Comment:
It is better to call the specific classes you need in `engine.entities` rather than calling all of `entities`